x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/utils.ts

```diff
@@ -31,15 +31,64 @@ export const getRedirectToAfterEngineCreation = ({
   return enginePath;
 };
 
+// Needed for string equality in tests
+export const removeWhitespace = (heredoc: string) => heredoc.replace(/\s+/g, ' ');
```

Comment:
We use `dedent` for these kind of things, please replace this if you can. Let me know if you need more info